docs/source/gbd2019_models/risk_risk_correlation/child_growth_risks/index.rst

```diff
@@ -104,21 +104,106 @@ Wasting and Stunting
+
+Notably, in GBD, there are LBWSG risk exposures at birth and among the early and late neonatal age groups only. For the child growth failure risks (wasting, stunting, and underweight), there are risk exposures for all age groups under five years, although risk effects for these risks apply to the post-neonatal and 1-4 age groups only.
+
+Links to documentation for relevant risk exposure pages include:
+
+- :ref:`GBD 2019 Low birthweight short gestation risk exposure <2019_risk_exposure_lbwsg>`
+
+- :ref:`GBD 2020 Wasting risk exposure <2020_risk_exposure_wasting_state_exposure>`
+
+- :ref:`GBD 2020 Stunting risk exposure <2020_risk_exposure_child_stunting>`
+
+Vivarium Modeling Strategy
+----------------------------
+
+The strategy for modeling risk-risk correlations related to child anthropometry in this document was developed for the needs of the :ref:`acute malnutrition treatment and prevention simulation <2019_concept_model_vivarium_ciff_sam>`. Different strategies may be more appropriate for different project needs and should be reavaluated when necessary.
+
 Birthweight and Wasting
 ++++++++++++++++++++++++
 
+In GBD, the birthweight exposure distribution is tracked through the late neonatal age group (28 days after birth) and affects the risk of mortality during this period. For the :ref:`acute malnutrition treatment and prevention simulation <2019_concept_model_vivarium_ciff_sam>`, child wasting exposures are modeled as a dynamic transition model (:ref:`documented here <2020_risk_exposure_wasting_state_exposure>`) rather than a standard static propensity-based risk exposure model (:ref:`such as the stunting risk exposure model <2020_risk_exposure_child_stunting>`). Therefore, there will be two approaches to modeling the correlation and causal association between birthweight and wasting, including 1) the initialization of wasting risk exposure upon transition into the postneonatal age group based on a simulant's birthweight exposure value, and 2) the correlation between a simulant's propensity for wasting incidence over time ("x-factor") with their birth weight exposure.
+
+.. todo::
+
+   Link to x-factor documentation for wasting model when it is developed
+
 Correlation
 ^^^^^^^^^^^^
 
+Initialization
+~~~~~~~~~~~~~~~
+
+When simulants are initialized into the early or late neonatal age groups or born into the simulation, they will be assigned a birthweight exposure value. When they transition into the postneonatal age group, they should be initialized into a particular state in the wasting model (e.g. TMREL, mild wasting, moderate wasting, or severe wasting) *based on their birthweight percentile within their overall location- and sex-specific birthweight distribution among the late neonatal age group.*
```

Comment:
I thought simulants were initialized into a state of the wasting model at birth, and it has RR 1.0 before post-neonatal age

Reply:
Yes, we were thinking that because we have the coefficient for one month instead of zero months that we would enforce it upon transition to the postneonatal age group since it won't have an effect on morbidity/mortality before that anyway, but now I am realizing that infants 0-6 months are eligible for wasting treatment, so we may want to continue to model wasting exposures in those groups too. @Ninicorn, do you agree?

This means that we would enforce the correlation (and causal impact) of birthweight on wasting *at birth* rather than one month and then track exposures accordingly. This is likely a conservative estimate, because I would expect the correlation to decrease over time (by the time wasting has an impact on outcomes) without enforcing a correlation with the x-factor, but I think I'll want to play around with it in a nano sim too to make sure.

Reply:
yes...we want to model wasting exposure from 1 month - 6months. Under 1 months don't have any burden due to wasting so we don't necessarily need to model it in those groups...but it messed up our model when we didn't have 'birth' prevalence. So yes agreed, we can enforce correlation at birth.

Reply:
Ok! Thanks, I will make this update.